.gitignore

```diff
@@ -8,5 +8,8 @@
 
 npm-debug.log*
 
+# tests
+/test-app
```

Comment:
maybe this one is specific to the `cli-tool` and hence should reside in its `.gitignore`, what do you think?

Reply:
At first I added it in the .gitignore of the `cli-tool`. But then I had to move the project generation into the root folder of this repository. So IMO it is more contextual to have it here. 
If we move it in the `.gitignore` of the cli folder, then it will have to be like `../../test-app` which feels wrong... What do you think? :)

Reply:
it's okay, I ran the generate command from the `cli-tools` folder locally, but as it can be run from the root directory also, I think it makes sense to keep it at the root.